src/app/tabs/test-tab.js

```diff
@@ -239,12 +245,18 @@ module.exports = class TestTab extends ViewPlugin {
   }
 
   updateRunAction (currentFile) {
-    let el = yo`<button id="runTestsTabRunAction" class="${css.runButton} btn btn-primary"  onclick="${this.runTests.bind(this)}">Run Tests</button>`
+    let el = yo`
+      <div id="runTestsTabRunAction" class="w-50 btn btn-primary"  onclick="${this.runTests.bind(this)}">
+        <span class="fas fa-play ml-2"></span>
+        <label class="btn p-1 ml-2 m-0">Run</label>
+      </div>
+    `
     const isSolidityActive = this.appManager.actives.includes('solidity')
     if (!currentFile || !isSolidityActive) {
       el.setAttribute('disabled', 'disabled')
-      if (!currentFile) el.setAttribute('title', 'No file selected')
-      if (!isSolidityActive) el.setAttribute('title', 'The solidity module should be activated')
+      if (!currentFile) {
+        el.setAttribute('title', 'No file selected')
+      } else if (!isSolidityActive) el.setAttribute('title', 'The solidity module should be activated')
```

Comment:
Better message can be: `'Solidity Compiler' plugin should be activated`

Reply:
it is like that in my local changes. I'll push them